adserver/forms.py

```diff
@@ -214,3 +218,45 @@ class Meta:
             "image": forms.FileInput(),
             "ad_types": forms.CheckboxSelectMultiple(),
         }
+
+
+class PublisherSettingsForm(forms.ModelForm):
+
+    """Form for letting publishers control publisher specific settings."""
+
+    def __init__(self, *args, **kwargs):
+        """Add the form helper and customize the look of the form."""
+        super().__init__(*args, **kwargs)
+
+        self.helper = FormHelper()
+        self.helper.layout = Layout(
+            Fieldset(
+                _("Control advertiser campaign types"),
+                "allow_paid_campaigns",
+                "allow_affiliate_campaigns",
+                "allow_community_campaigns",
+                "allow_house_campaigns",
+                HTML(
+                    "<p class='form-text small text-muted'>"
+                    + str(
+                        _(
+                            "Use these checkboxes to control the types of advertiser campaigns "
+                            "that are allowed on your site. "
+                            "House and community campaigns can be especially useful when settings things up."
+                        )
+                    )
+                    + "</p>"
+                ),
+                css_class="my-3",
+            ),
+            Submit("submit", "Save settings"),
+        )
+
+    class Meta:
+        model = Publisher
+        fields = [
+            "allow_paid_campaigns",
```

Comment:
Yea -- I'd say this shouldn't be a boolean, but should probably just show the status, and let us change it.

Reply:
Just so we're clear, you're thinking something like `Publisher.status` and the values are "approved" and "not approved"? Would a boolean `Publisher.approved` be better?

Reply:
I think `allow_paid_campaigns` is fine for now, and we can treat it like `approved` as something that we control. Seems fine to me.

Reply:
Sorry -- my initial comment was confusing. I just meant that we shouldn't let the user change it, but it should be something only we can change.

Reply:
Sounds good. I'll remove it from the form and we should be good to go.